src/lib.rs

```diff
@@ -1784,6 +1784,9 @@ impl Build {
         if cfg!(target_os = "macos") {
             self.fix_env_for_apple_os(&mut cmd)?;
         }
+        if msvc {
```

Comment:
Why only when linking for MSVC? Seems like we'd want to do this everywhere, if we wanted to do it in the first place?

Reply:
Thanks updated.